src/Plugin/media/Source/GoogleForm.php

```diff
@@ -26,6 +31,31 @@ class GoogleForm extends MediaSourceBase implements MediaSourceFieldConstraintsI
    */
   const METADATA_ATTRIBUTE_ID = 'id';
 
+  /**
+   * The name of the height field.
+   *
+   * @var string
+   */
+  protected $heightField;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entity_type_manager, EntityFieldManagerInterface $entity_field_manager, FieldTypePluginManagerInterface $field_type_manager, ConfigFactoryInterface $config_factory) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition, $entity_type_manager, $entity_field_manager, $field_type_manager, $config_factory);
+    $configuration = $this->getConfiguration();
+    $this->heightField = $configuration['height_field_name'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [
+      'height_field_name' => 'field_media_google_form_hgt',
```

Comment:
This produces an error if the field doesn't exist.
```suggestion
      'height_field_name' => NULL,
```

Reply:
![image](https://user-images.githubusercontent.com/7185045/101077314-87b3c880-3559-11eb-8aa0-4a8fcfa177db.png)